assets/rich_text_components_definitions.ts

```diff
@@ -145,7 +145,14 @@ export = {
     "is_lesson_related": true,
     "is_block_element": false,
     "customization_arg_specs": [{
-      "name": "skill_summary",
+      "name": "text",
+      "description": "Enter the link text to be displayed",
```

Comment:
"The text to be displayed on the skill reference link

Reply:
Same comment.

Reply:
Yeah, done, renamed it to the text in the previous comment.